### DISCUSSION TITLE
Ring Doorbell 2 Livestream + Events

### DISCUSSION THREAD

**Discussion by Shimor1** [tsightler/ring-mqtt#398] [Support Q&A]
Firstly, Home Assistant does not detect any live camera entity, only the snapshot one. And this snapshot entity does not seem to work properly: I set the snapshot mode as motions (which are detected properly) but the image does not update in this mode (see below)

![image](https://user-images.githubusercontent.com/70377962/178141145-b3105622-b354-4990-bca9-f7caf327e951.png)

**Comment by tsightler** [comment]
Regarding live cameras, please read the documentation as it is quite clear that live cameras will need to be added manually, including providing the steps required.

For motion snapshots, I'm not really sure what you think I can tell from that screenshot, but the addon provides detailed logs for all events including received motion events and snapshot updates.  The logs from such an event would go a long way to telling what is happening in your case.  Note that, if you have a battery/low-power mode camera, motion snapshots will require a Ring Protect plan with the rich notifications feature enabled.
